tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+    ...    ${PUBLIC_API_FILES_DIR}
+
+Add data guidance to subjects
+    user clicks link    Data and files
+    user waits until h2 is visible    Add data file to release
+
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_1_NAME}
+
+    user enters text into data guidance data file content editor    ${SUBJECT_1_NAME}
+    ...    ${SUBJECT_1_NAME} Main guidance content
+
+    user clicks button    Save guidance
+
+Create 1st API dataset
+    user scrolls to the top of the page
+    user clicks link    API data sets
+    user waits until h2 is visible    API data sets
+
+    user clicks button    Create API data set
+    ${modal}=    user waits until modal is visible    Create a new API data set
+    user chooses select option    id:apiDataSetCreateForm-releaseFileId    ${SUBJECT_1_NAME}
+    user clicks button    Confirm new API data set
+
+    user waits until page finishes loading
+    user waits until modal is not visible    Create a new API data set
+
+User waits until the 1st API dataset status changes to 'Ready'
+    user waits until h3 is visible    Draft version details
+    wait until keyword succeeds    10x    %{WAIT_SMALL}s    Verify status of API Datasets    Ready
+
+Add headline text block to Content page
+    user clicks link    Back to API data sets
+    user navigates to content page    ${PUBLICATION_NAME}
+    user adds headlines text block
+    user adds content to headlines text block    Headline text block text
+
+Approve first release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+Create a second draft release via api
+    user navigates to publication page from dashboard    ${PUBLICATION_NAME}
+    user creates release from publication page    ${PUBLICATION_NAME}    Academic year    3010
+
+# In this new subject, the "School type" filter has been removed entirely.
+
+Upload subject to second release
+    user uploads subject and waits until complete    ${SUBJECT_2_NAME}    absence_school_major_auto.csv
+    ...    absence_school_major_auto.meta.csv    ${PUBLIC_API_FILES_DIR}
+
+Add data guidance to second release
+    user clicks link    Data and files
+    user waits until h2 is visible    Add data file to release
+
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_2_NAME}
+
+    user enters text into data guidance data file content editor    ${SUBJECT_2_NAME}
+    ...    ${SUBJECT_2_NAME} Main guidance content
+
+    user clicks button    Save guidance
+
+Create a different version of an API dataset with major changes
```

Comment:
`data set` instead of `dataset`

Reply:
Another hangover from the pre-existing suite - I have renamed all "dataset" to "data set" in the original suite and the new one too